ui/page/top/view.jsx

```diff
@@ -31,23 +35,38 @@ function TopPage(props: Props) {
           </span>
         )}
         header={
-          <div className="claim-search__menu-group">
-            <Button
-              label={name}
-              button="alt"
-              onClick={() => setChannelActive(false)}
-              className={classnames('button-toggle', {
-                'button-toggle--active': !channelActive,
-              })}
-            />
-            <Button
-              label={`@${name}`}
-              button="alt"
-              onClick={() => setChannelActive(true)}
-              className={classnames('button-toggle', {
-                'button-toggle--active': channelActive,
-              })}
-            />
+          <div className="claim-search__menu-group--between">
```

Comment:
This doesn't need to be a new style. You can keep the `header` prop as it was, and pass the action text in with the `meta` prop.

https://github.com/lbryio/lbry-desktop/blob/master/ui/page/channelsFollowing/view.jsx#L35